scripts/installGit.ps1

```diff
@@ -41,7 +41,7 @@ try {
 
         Set-Location \ | Out-Host
         # cmd /C git clone https://github.com/robe070/cookbooks.git $GitRepo '2>&1'
-        Run-ExitCode 'git' @('clone', 'https://github.com/robe070/cookbooks.git', $GitRepo) | Out-Host
+        Run-ExitCode 'git' @('clone', 'https://github.com/praveen-acc/cookbooks.git', $GitRepo) | Out-Host
```

Comment:
Again, another personalised setting that does not belong in the script. This must be made into a parameter to `bake-ide-ami.ps1` and set a `$script:gitrepo` variable and use that in `installGit.ps1`